test/blackbox-tests/test-cases/coq/coqtop/coqtop-flags.t/run.t

```diff
@@ -26,7 +26,10 @@ The flags passed to coqc:
   -I lib/coq/../coq-core/plugins/ssreflect
   -I lib/coq/../coq-core/plugins/ssrmatching
   -I lib/coq/../coq-core/plugins/tauto
-  -I lib/coq/../coq-core/plugins/tutorial
+  -I lib/coq/../coq-core/plugins/tutorial/p0
+  -I lib/coq/../coq-core/plugins/tutorial/p1
+  -I lib/coq/../coq-core/plugins/tutorial/p2
+  -I lib/coq/../coq-core/plugins/tutorial/p3
```

Comment:
I've added this for consistency with what Coq does.

Coq also adds `$opam_root/lib` but I'd rather avoid this for now.

Reply:
We should make a bug upstream about this. For now it makes sense to keep the scanning behavior close.